1.1/geo.ttl

```diff
@@ -580,6 +589,53 @@ geosparql:
 	skos:prefLabel "spatial dimension"@en ;
 .
 
+:metricSpatialResolution
+	a owl:DatatypeProperty ;
+	rdfs:domain :Geometry ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The spatial resolution of a Geometry in meters."""@en ;
+	skos:note """Spatial resolution specifies the level of detail of a Geometry. It the smallest dinstinguishable distance between spatially adjacent coordinates."""@en;
+	skos:prefLabel "metric spatial resolution"@en ;
+.
+
+:metricSpatialAccuracy
+	a owl:DatatypeProperty ;
+	rdfs:domain :Geometry ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The positional accuracy of the coordinates of a Geometry in meters."""@en ;
+	skos:note """Spatial accuracy is applicable when a Geometry is used to represent a Feature. It is expressed as a distance that indicates the truthfullness of the positions (coordinates) that define the Geometry. In this case accuracy defines a zone surrounding each coordinate within wich the real positions are known to be. The accuracy value defines this zone as a distance from the coordinate(s) in all directions (e.g. a line, a circle or a sphere, depending on spatial dimension)."""@en;
+	skos:prefLabel "metric spatial accuracy"@en ;
+.
+
+:metricLength
+	a owl:DatatypeProperty ;
+	rdfs:domain :Feature ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The length of a Feature in meters."""@en ;
+	skos:prefLabel "metric length"@en ;
+.
+
+:metricArea
+	a owl:DatatypeProperty ;
+	rdfs:domain :Feature ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The area of a Feature in square meters."""@en ;
+	skos:prefLabel "metric area"@en ;
+.
+
+:metricVolume
+	a owl:DatatypeProperty ;
+	rdfs:domain :Feature ;
+	rdfs:range xsd:double ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	skos:definition """The volume of a Feature in cubic meters."""@en ;
+	skos:prefLabel "metric volume"@en ;
```

Comment:
"volume in cubic meters" - in vernacular English 'metric' denotes the whole system of units, not just metres.